Eat_and_Greet/src/comps/UserInformation/UserInformation.tsx

```diff
@@ -3,16 +3,15 @@ import { Image, ScrollView, Text, View, StyleSheet, TouchableOpacity, Alert } fr
 import CircularIcon from '../CircularIcon/CircularIcon';
 import styles from '../../style';
  import UserManager from '../../logic/UserManager';
```

Comment:
I know you mentioned we may have to change the name to UserManager from userManager, but I'm confused because the file name itself seems to still be userManager